packages/devtools_app/lib/src/primitives/feature_flags.dart

```diff
@@ -14,12 +14,17 @@
 ///   ]
 const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments');
 
+/// If true, features, ready for beta testing, will be on.
+///
+/// See [_kEnableExperiments] documentation  for usage.
+const bool _kEnableBeta = bool.fromEnvironment('enable_beta');
```

Comment:
instead of using an environment variable, can we just default this value to !isExternalBuild? g3 users won't be running DevTools from the source code where they can pass in arguments